package.json

```diff
@@ -20,7 +20,6 @@
 		"ci:build": "npm run prod",
 		"ci:test": "npm run test-js",
 		"dev": "node scripts/watch.js",
-		"generate": "graphql-codegen",
```

Comment:
this script isn't needed, it was used only once to generated type definition from server-side schema, but then we discussed that we want to do that manually